assets/js/serviceworker.js

```diff
@@ -0,0 +1,90 @@
+var CACHE_NAME = 'sass-guidelines-cache';
+// The files we want to cache
+var urlsToCache = [
```

Comment:
I'm not very good with Node, so I'm not going to do it, but we could parse the `offline.appcache` and simply take that list?